app/graphql/mutations/invoices/retry_tax_provider_voiding.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Invoices
+    class RetryTaxProviderVoiding < BaseMutation
+      include AuthenticableApiUser
+      include RequiredOrganization
+
+      REQUIRED_PERMISSION = 'invoices:update'
+
+      description 'Retry voided invoice sync'
+
+      argument :id, ID, required: true
+
+      type Types::Invoices::Object
+
+      def resolve(**args)
+        invoice = current_organization.invoices.visible.find_by(id: args[:id])
```

Comment:
should we use `visible` here?
voiding a `draft` or already `voided` invoice does not make sense, right?

Reply:
`visible` also includes `voided`, `draft` and `generating` are the only ones excluded from `visible`